src/renderer.js

```diff
@@ -159,25 +159,36 @@ document.addEventListener("DOMContentLoaded", function () {
   var calendar = new FullCalendar.Calendar(calendarEl, {
     initialView: "dayGridMonth",
     dateClick: function (info) {
-      // Use the load dates for the current and day after the current day here
-      monthNumber = info.dateStr[5] + info.dateStr[6];
-      dayNumber = info.dateStr[8] + info.dateStr[9];
-      loadCurrentDayTasks(dayNumber, monthNumber);
+      const clickedDate = new Date(info.dateStr);
+      const clickedDay = clickedDate.getDate();
+      const clickedMonth = clickedDate.getMonth() + 1;
+
+      const tomorrowDay = clickedDate.getDate();
+      const tommorowMonthmonth = clickedDate.getMonth();
+      const tommorowYear = clickedDate.getFullYear();
+      const tomorrowDate = new Date(
+        tommorowYear,
+        tommorowMonthmonth,
+        tomorrowDay + 1
+      );
+      loadCurrentDayTasks(clickedDay, clickedMonth);
+      loadTomorrowTasks(clickedDate, tomorrowDate);
     },
   });
   calendar.render();
 });
-function loadCurrentDayTasks(dayNumber, monthNumber) {
+function loadCurrentDayTasks(clickedDay, clickedMonth) {
   const existingTasks = JSON.parse(localStorage.getItem("tasks")) || [];
 
   // Clears the tasklist
   tasksList.innerHTML = "";
 
   existingTasks.forEach((task) => {
-    if (
```

Comment:
Using dates instead of string helped make these logical expressions muuuuchh more readable and simple to work with.